project/NativeImage.scala

```diff
@@ -74,14 +74,7 @@ object NativeImage {
       val debugParameters =
         if (includeDebugInfo) Seq("-H:GenerateDebugInfo=1") else Seq()
 
-      val (staticParameters, pathExts) =
-        if (staticOnLinux && Platform.isLinux) {
-          // Note [Static Build On Linux]
-          val buildCache =
-            subProjectRoot / "build-cache"
-          val path = ensureMuslIsInstalled(buildCache, log)
-          (Seq("--static", "--libc=musl"), Seq(path.toString))
```

Comment:
According to https://github.com/oracle/graal/issues/4028 it is not possible to load `.so` files with `musl` - using regular `libc`.